package.json

```diff
@@ -40,31 +40,31 @@
     "node": ">=7"
   },
   "dependencies": {
-    "aws-sdk": "^2.586.0",
+    "aws-sdk": "^2.669.0",
     "dynamodb-data-types": "^3.0.1",
     "dynamodb-update-expression": "^0.1.21",
     "dynamodb-wrapper": "^1.4.1",
-    "ramda": "^0.25.0"
+    "ramda": "^0.27.0"
   },
   "devDependencies": {
-    "@commitlint/config-conventional": "^7.1.2",
-    "commitlint": "^7.1.2",
-    "docdash": "^1.0.0",
-    "eslint": "^5.6.0",
-    "eslint-config-prettier": "^3.1.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-node": "^7.0.1",
-    "eslint-plugin-prettier": "^2.6.2",
+    "@commitlint/config-conventional": "^8.3.4",
+    "commitlint": "^8.3.5",
+    "docdash": "^1.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-ramda": "^2.5.1",
     "eslint-plugin-standard": "^4.0.0",
-    "husky": "^1.0.0-rc.15",
-    "jest": "^23.6.0",
-    "jsdoc": "^3.5.5",
-    "lint-staged": "^7.3.0",
+    "husky": "^4.2.5",
+    "jest": "^26.0.1",
+    "jsdoc": "^3.6.4",
+    "lint-staged": "^10.2.2",
     "npm-run-all": "^4.1.3",
-    "prettier": "^1.14.3"
+    "prettier": "^2.0.5"
```

Comment:
Mind also updating `prettierrc` to:

```json
{
  "printWidth": 100,
  "singleQuote": true,
  "arrowParens": "avoid",
  "trailingComma": "none"
}
```

...and re-formatting sources (`yarn prettier`), please?